src/labels_by_user_input.py

```diff
@@ -1,21 +1,42 @@
+import sys
+
 from src.utils import get_labels
 
 
-LABEL_VERIFIED = "Verified"
+LABELS_SET = {
+    "hold": "do-not-merge",
+    "verified": "Verified",
+    "bug": "bug",
+}
+
+
+def remove_label(pull, label):
+    print(f"Remove {label} from {pull.title}")
+    pull.remove_from_labels(label)
 
 
 def remove_verified_label(pull):
     labels = get_labels(pull=pull)
-    if LABEL_VERIFIED in labels:
-        print(f"Remove {LABEL_VERIFIED} from {pull.title}")
-        pull.remove_from_labels(LABEL_VERIFIED)
+    if LABELS_SET["verified"] in labels:
+        remove_label(pull=pull, label=LABELS_SET["verified"])
 
 
 def labels_by_user_input(data, pull):
     body = data["comment"]["body"]
-    if "/verified" in body and LABEL_VERIFIED not in get_labels(pull=pull):
-        print(f"Adding {LABEL_VERIFIED} to {pull.title}")
-        pull.add_to_labels(LABEL_VERIFIED)
+    labels = get_labels(pull=pull)
+
+    if body[0] == "/":
+        label = body[1:].lower()
+
+        try:
+
+            if label[:2] == "un" and LABELS_SET[label[2:]] in labels:
+                remove_label(pull=pull, label=LABELS_SET[label[2:]])
+
+            elif label not in labels:
+                print(f"Adding {label} to {pull.title}")
+                pull.add_to_labels(LABELS_SET[label])
 
-    if "/unverified" in body:
-        remove_verified_label(pull=pull)
+        except KeyError:
```

Comment:
no need to except KeyError, we know what we have in our dict.

I prefers to add a comment in the PR about unsupported labels when the user try to use one.

Reply:
How about keeping the 'except', but using it for comments?
Or do you reckon I should look for a way to do it in the workflow yml?